tasks/8.js

```diff
@@ -0,0 +1,26 @@
+function spy(fn) {
+  let count = 0;
+  return function wrap() {
+    count++;
+    wrap.report = function () {
+      return fn(count);
```

Comment:
The idea would be for this function to return `{ totalCalls: count }` and it should matter what you pass into the `spy` function.  

Also, in this case your implementation file handles half of the testing of the solution, only `spy` should be exported. 😉

Reply:
What do you mean 

> it should matter what you pass into the spy function.

`spied(2);
spied(100)`

should return 102 not 2?

Reply:
Sorry, my bad, too late review. Should have been `shouldn't` matter. 🙈